### DISCUSSION TITLE
The xopp icon in Ubuntu

### DISCUSSION THREAD

**Discussion by Soothfast** [xournalpp/xournalpp#5282] [Q&A]
Yesterday I got my first Ubuntu laptop (a Lenovo).  Trying to banish Windows from my life.  My experience with any kind of Linux operating system is now approaching 6 hours.   I've used Xournal++ for almost 2 years in Windows, however, and yesterday I got it installed on the Ubuntu system, and have it working nicely with a Huion writing tablet (Kamvas 16).

Really, everything is just fine, except that the xopp files all have the gzip icon.  I have a test file called test.xopp on the desktop, so near as I can tell it really is a xopp file, but it has the gzip icon.  Now, I read somewhere that a xopp file is actually...let me find it...here: "xopp is a gzipped XML file."  So I don't know how that works.  It's an xopp file, with file extension .xopp, but it's really a gzip file, despite it not having a .gz file extension.  Uh...

When I click on this test.xopp "entity" on the desktop, it does not open a folder, it opens the file like a normal xopp file, in Xournal++.  Fine.

So here's my big dumb question:  is there any way to get all xopp files in my Ubuntu system to have the traditional xopp icon?  You know, the pencil writing the letter x thing.  "Explanations" I've found elsewhere for globally changing a file type's icon in Ubuntu seem to require a BS degree in computer science to follow.

**Comment by rolandlo** [comment]
How did you install Xournal++? From the software center? Is it the snap version, the deb version or the flatpak?

**Reply by Soothfast** [reply]
Hi,

I'm almost certain the snap version, from here:
https://snapcraft.io/xournalpp

Version 1.2.1
GTX version 3.24.38
Git commit b48932a2 (not sure what this is, but in case it's useful)
